index.js

```diff
@@ -7,6 +7,9 @@ const jsonpatch = require('fast-json-patch');
 const request = require('request');
```

Comment:
I think casing consistency matters so in this case should `jsonpatch` be `jsonPatch` instead?

Reply:
hum i don't think it makes a difference since we don't use the latter anywhere that i'm aware of and the module itself uses all lowercase in examples https://www.npmjs.com/package/fast-json-patch

Reply:
AH! ok that makes sense - i can definitely change that